test-framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -161,6 +161,25 @@ public static void restoreContentType() {
         Requests.INDEX_CONTENT_TYPE = XContentType.JSON;
     }
 
+    private static boolean TESTS_SEED_PROPERTY_SET = false;
+    @BeforeClass
+    @SuppressForbidden(reason = "sets tests.seed")
```

Comment:
no i do not. but this IDE cannot compromise the actual build, which is 'gradle check'. changing tests.seed in this way can compromise the build, because then the values for other things looking for this (such as lucene) depends on class initialization order.